src/test/njs_unit_test.c

```diff
@@ -16882,6 +16912,115 @@ static njs_unit_test_t  njs_test[] =
     { njs_str("Date.constructor === Function"),
       njs_str("true") },
 
+    { njs_str("var y = Object.defineProperty({}, Symbol.toPrimitive, {"
+              "    get: function() {"
+              "        throw new Test262Error();"
```

Comment:
Test262Error() is not defined. It is better to use local and more specific exceptions.
See here https://github.com/xeioex/njs/commit/6cedc55f761fc1afd340da22bd0fbe7b23f91521.